dl4j-labs/src/main/java/ai/certifai/solution/earlyStopping/EarlyStopping.java

```diff
@@ -0,0 +1,257 @@
+
+        TransformProcess tp = new TransformProcess.Builder(sc)
+                .categoricalToInteger("Class")
+                .build();
+
+//        Checking the schema
+        Schema outputSchema = tp.getFinalSchema();
+        System.out.println(outputSchema);
+
+
+        List<List<Writable>> allData = new ArrayList<>();
+
+        while(rr.hasNext()){
+            allData.add(rr.next());
+        }
+
+        List<List<Writable>> processData = LocalTransformExecutor.execute(allData, tp);
+
+//========================================================================
+        //  Step 3 : Create Iterator ,splitting trainData and testData
+//========================================================================
+
+        //Create iterator from process data
+        CollectionRecordReader collectionRR = new CollectionRecordReader(processData);
+
+        //Input batch size , label index , and number of label
+        DataSetIterator dataSetIterator = new RecordReaderDataSetIterator(collectionRR, processData.size(),8,numClass);
+
+//        //Create Iterator and shuffle the dat
+        DataSet fullDataset = dataSetIterator.next();
+        fullDataset.shuffle(seed);
+//
+//        //Input split ratio
+        SplitTestAndTrain testAndTrain = fullDataset.splitTestAndTrain(splitRatio);
+//
+        //Get train and test dataset
+        DataSet trainData = testAndTrain.getTrain();
+        DataSet testData = testAndTrain.getTest();
+
+        //printout size
+        System.out.println("Training vector : ");
+        System.out.println(Arrays.toString(trainData.getFeatures().shape()));
+        System.out.println("Test vector : ");
+        System.out.println(Arrays.toString(testData.getFeatures().shape()));
+
+//========================================================================
+        //  Step 4 : DataNormalization
+//========================================================================
+
+        //Data normalization
+        DataNormalization normalizer = new NormalizerMinMaxScaler();
+        normalizer.fit(trainData);
+        normalizer.transform(trainData);
+        normalizer.transform(testData);
+
+//========================================================================
+        //  Step 5 : Network Configuration
+//========================================================================
+
+        //Get network configuration
+        MultiLayerConfiguration modelConfig = getConfig(numInput, numClass, learningRate);
+
+        //Define network
+        MultiLayerNetwork model = new MultiLayerNetwork(modelConfig);
+
+
+//========================================================================
+        //  Step 6 : Early Stopping Configuration
+//========================================================================
+
+       EarlyStoppingConfiguration esConfig = new EarlyStoppingConfiguration.Builder()
+               .epochTerminationConditions(new MaxEpochsTerminationCondition(epoch))
```

Comment:
add comments to explain these function
`
.epochTerminationConditions()
.scoreCalculator()
.evaluateEveryNEpochs()
`